[0m16:29:22.821724 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A1D87FB050>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A1D87F8990>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A1D87FB610>]}


============================== 16:29:22.849468 | cadab24a-20b1-4dc3-865f-790ebc30a565 ==============================
[0m16:29:22.849468 [info ] [MainThread]: Running with dbt=1.8.4
[0m16:29:22.851934 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\hp\\.dbt', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'version_check': 'True', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'invocation_command': 'dbt init', 'introspect': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m16:29:22.980816 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': WindowsPath('C:/Users/hp/.dbt')}
[0m16:29:22.982801 [info ] [MainThread]: Creating dbt configuration folder at 
[0m16:29:57.252720 [debug] [MainThread]: Starter project path: C:\Users\hp\AppData\Local\Programs\Python\Python311\Lib\site-packages\dbt\include\starter_project
[0m16:29:57.400239 [info ] [MainThread]: 
Your new dbt project "dbt_project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m16:29:57.405276 [info ] [MainThread]: Setting up your profile.
[0m16:35:42.218549 [info ] [MainThread]: Profile dbt_project written to C:\Users\hp\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m16:35:42.407415 [debug] [MainThread]: Command `dbt init` succeeded at 16:35:42.397199 after 379.81 seconds
[0m16:35:42.431741 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A1D880F8D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A1D880F990>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A1D3505590>]}
[0m16:35:42.461545 [debug] [MainThread]: Flushing usage events
[0m22:49:08.635409 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025B3FDEF710>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025B3FD8F250>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025B3FDED990>]}


============================== 22:49:08.659418 | 43b1447f-739e-458f-a691-95b520b52d6a ==============================
[0m22:49:08.659418 [info ] [MainThread]: Running with dbt=1.8.4
[0m22:49:08.663422 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'version_check': 'True', 'profiles_dir': 'C:\\Users\\hp\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt parse', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m22:49:08.668368 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\hp\Desktop\dag\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m22:49:08.675915 [debug] [MainThread]: Command `dbt parse` failed at 22:49:08.675915 after 0.19 seconds
[0m22:49:08.675915 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025B3FDF61D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025B3FD95750>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025B3FDEEF10>]}
[0m22:49:08.679922 [debug] [MainThread]: Flushing usage events
[0m22:54:54.493237 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FCC8866F50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FCC8865010>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FCC8865510>]}


============================== 22:54:54.509689 | f7e06db1-c7a7-4066-b1b8-d3e3499d6673 ==============================
[0m22:54:54.509689 [info ] [MainThread]: Running with dbt=1.8.4
[0m22:54:54.515331 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\hp\\.dbt', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'version_check': 'True', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'invocation_command': 'dbt parse', 'introspect': 'True', 'static_parser': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m22:54:54.517236 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\hp\Desktop\dag\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m22:54:54.525780 [debug] [MainThread]: Command `dbt parse` failed at 22:54:54.525780 after 0.38 seconds
[0m22:54:54.525780 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FCC8814B90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FCC8871610>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FCC88723D0>]}
[0m22:54:54.525780 [debug] [MainThread]: Flushing usage events
[0m22:56:00.611047 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020E5F49F510>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020E5DB66590>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020E5F4DBAD0>]}


============================== 22:56:00.643215 | 26841aef-e3d5-4027-934e-9c544bd9d4dc ==============================
[0m22:56:00.643215 [info ] [MainThread]: Running with dbt=1.8.4
[0m22:56:00.647267 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\hp\\.dbt', 'debug': 'False', 'version_check': 'True', 'log_path': 'C:\\Users\\hp\\Desktop\\dag\\logs', 'fail_fast': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt run', 'log_format': 'default', 'introspect': 'True', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m22:56:05.499846 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'project_id', 'label': '26841aef-e3d5-4027-934e-9c544bd9d4dc', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020E702F1810>]}
[0m22:56:05.631849 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'adapter_info', 'label': '26841aef-e3d5-4027-934e-9c544bd9d4dc', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020E5E0AABD0>]}
[0m22:56:05.635845 [info ] [MainThread]: Registered adapter: snowflake=1.8.3
[0m22:56:05.651847 [debug] [MainThread]: checksum: 54398663eddc8e4ac172fdab397e09b2aa6984947a84e6ec7983096c3fae7b7a, vars: {}, profile: , target: , version: 1.8.4
[0m22:56:05.655858 [info ] [MainThread]: Unable to do partial parsing because saved manifest not found. Starting full parse.
[0m22:56:05.659857 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'partial_parser', 'label': '26841aef-e3d5-4027-934e-9c544bd9d4dc', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020E5F3253D0>]}
[0m22:56:08.323956 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'load_project', 'label': '26841aef-e3d5-4027-934e-9c544bd9d4dc', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020E5C964650>]}
[0m22:56:08.505924 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'resource_counts', 'label': '26841aef-e3d5-4027-934e-9c544bd9d4dc', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020E713F51D0>]}
[0m22:56:08.506791 [info ] [MainThread]: Found 448 macros
[0m22:56:08.506791 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'runnable_timing', 'label': '26841aef-e3d5-4027-934e-9c544bd9d4dc', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020E5DAC51D0>]}
[0m22:56:08.510304 [info ] [MainThread]: 
[0m22:56:08.510304 [warn ] [MainThread]: Nothing to do. Try checking your model configs and model specification args
[0m22:56:08.514325 [debug] [MainThread]: Command end result
[0m22:56:08.578471 [debug] [MainThread]: Command `dbt run` succeeded at 22:56:08.578471 after 8.13 seconds
[0m22:56:08.582531 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020E5A1B0D10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020E5A1E9110>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020E5A1E9CD0>]}
[0m22:56:08.582531 [debug] [MainThread]: Flushing usage events
